### PR TITLE
Remove interaction with body to prevent scrolling when meganav is open

### LINKED ISSUE #129: Stop adding CSS to <body>
Currently, to stop scrolling in the small screen view, we're adding 

```css
overflow-y: hidden;
height: 100%;
```

to the body element with a CSS class that's applied via JS

### PR DESCRIPTION
**Pull request by onishiweb**
Changed this to add `position:fixed` to the header element with `width` and `height` 100% to prevent scrolling. I've tested it across a variety of browsers including Android, iOS, and Windows and all seems OK.

@wheresrhys Alberto mentioned there might have been an issue with this in the past, let me know if you remember anything.

Closes #129

### CODE REVIEW

File: src/scss/_nav.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @if $o-header-is-silent == false {
 
-	// This is applied to the <body> via JS, this does not comply
-	// with the Origami spec, and issue #129 has been raised
+	// This is applied to the o-header element via JS
 	// It's used to avoid scrolling when the meganav
 	// is open in a mobile view
 	.o-header--mega-nav-open {
```
Comment (AlbertoElias) [inline src/scss/_nav.scss]: I think you might be able to ditch this class as there should be a different selector for an opened meganav container

**Comment by wheresrhys** [comment]
As a rule of thumb it's probably worth testing the header on the next homepage, as that's the most complex implementation of it on next. If it works there I doubt next would have a problem with it

File: src/scss/_nav.scss (hunk 1 above)
Comment (onishiweb) [inline src/scss/_nav.scss]: In that case do you mean adding the styles directly to the header element or just a different name of the class? Sorry, not sure I follow completely.

Comment (AlbertoElias) [inline src/scss/_nav.scss]: Oh, true! You're applying this to the header element, sorry! Ignore me. If as Rhys says, this works on Next, fine with me to release it :shipit:

**Comment by onishiweb** [comment]
Right, that was the issue why this wouldn't be a good fix. The stuff before the header means we can't just fix the header over the top and let it hide everything else.

**Pull request closed without merging**